incubator/model-core.spec/src/main/scripts/io/aklivity/zilla/specs/model/core/schema/double.schema.patch.json

```diff
@@ -0,0 +1,99 @@
+[
+    {
+        "op": "add",
+        "path": "/$defs/types/enum/-",
+        "value": "double"
+    },
+    {
+        "op": "add",
+        "path": "/$defs/converter/oneOf/1/allOf/-",
```

Comment:
Let's review the structure of the schema for converters and validators, this doesn't look quite right to me.

Also we need to register for both `converters` and `validators` enums, agree?